### PR TITLE
[DOCFIX] Improve troubleshooting doc.

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

**Comment by bf8086** [comment]
@aaudiber PTAL.

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
 value in `conf/alluxio-site.properties`. Note that, reducing the value of these two properties may
 potentially add latency for master to serve requests.
 
-TODO(feng) add a section about discover file updates in under fs.
+### Q: I added some files in under file system. How can I reveal the files in Alluxio?
+
+By default, Alluxio loads the list of files the first time a directory is visited. Alluxio will keep using the cached file list regardless of the changes in the under file system. To reveal new files from under file system, you can use the command `alluxio fs ls -f /some/path` to manually discover the new content inside a specific folder. Another way to refresh a directory is to use UFS sync.
```
Comment (aaudiber) [inline docs/en/operation/Troubleshooting.md]: can you wrap the lines more frequently, e.g. around 100 chars? Long lines like this are harder to work with in many editors, and also harder to review changes in.

File: docs/en/operation/Troubleshooting.md [hunk 2]
```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
...
+You can either use it in command line by running `alluxio fs ls -R -Dalluxio.user.file.metadata.sync.interval=${SOME_INTERVAL} /path` or by setting the same configuration property in masters' `alluxio-site.properties`. The value for the configuration property is used to
+determine the minimum interval between two syncs. You can read more about loading files from under file system [here](../advanced/Loading-Under-File-Storage-Metadata.html).
+
+
```
Comment (aaudiber) [inline docs/en/operation/Troubleshooting.md]: extra newline

File: docs/en/operation/Troubleshooting.md [hunk 3]
```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
...
+### Q: I see an error "Block ?????? is unavailable in both Alluxio and UFS" while reading some file. Where is my file?
```
Comment (aaudiber) [inline docs/en/operation/Troubleshooting.md]: this error is also shown when the block exists in the UFS, but there are no workers available (e.g. all workers have died)

File: docs/en/operation/Troubleshooting.md [hunk 4]
```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
...
-TODO(feng) add a section for hanging commands
+`THROUGH`: data will be only written to UFS
+
+By default the write type used by Alluxio client is `MUST_CACHE`, therefore a new file written to Alluxio is only stored in Alluxio
+worker storage, and can be evicted when Alluxio worker storage is full and some new data needs to be cached. To make sure data is persisted, either use `CACHE_THROUGH` or `THROUGH` write type, or [pin](../basic/Command-Line-Interface.html#pin) the files you would like to preserve.
+
+
+### Q: I run some Alluxio shell command, and it hangs without giving any output. What's going on?
+
+A: Most Alluxio shell commands requires connecting to Alluxio master to execute. If the command fails to connect to master it will keep retrying several times, appearing as "hanging" for a long time. It is also possible that some command can take a long time to execute, such as persisting a large file on a slow UFS. If you want to know what happens under the hood, check the user log(stored as `${ALLUXIO_HOME}/logs/user_${USER_NAME}.log` by default) or master log (stored as `${ALLUXIO_HOME}/logs/master.log` on the master node by default).
```
Comment (aaudiber) [inline docs/en/operation/Troubleshooting.md]: space between `log` and `(`

Comment (aaudiber) [inline docs/en/operation/Troubleshooting.md]: commands requires -> commands require

File: docs/en/operation/Troubleshooting.md (hunk 1 above)
Comment (bf8086) [reply docs/en/operation/Troubleshooting.md]: Done.

File: docs/en/operation/Troubleshooting.md (hunk 2 above)
Comment (bf8086) [reply docs/en/operation/Troubleshooting.md]: Removed.

File: docs/en/operation/Troubleshooting.md (hunk 3 above)
Comment (bf8086) [reply docs/en/operation/Troubleshooting.md]: Added a section to explain the alternative cause.

File: docs/en/operation/Troubleshooting.md (hunk 4 above)
Comment (bf8086) [reply docs/en/operation/Troubleshooting.md]: Added.

**Pull request merged**